README.md

```diff
@@ -110,19 +110,23 @@ per RFC 8152, are rejected by the go-cose library.
 
 ### Conformance Tests
 
-go-cose runs the [GlueCOSE](https://github.com/gluecose/test-vectors) test suite on every local `go test` execution.
+`go-cose` runs the [GlueCOSE](https://github.com/gluecose/test-vectors) test suite on every local `go test` execution.
 These are also executed on every CI job.
 
 ### Fuzz Tests
 
-go-cose implements several fuzz tests using [Go's native fuzzing](https://go.dev/doc/fuzz).
+`go-cose` implements several fuzz tests using [Go's native fuzzing](https://go.dev/doc/fuzz).
 
 Fuzzing requires Go 1.18 or higher, and can be executed as follows:
 
 ```bash
 go test -fuzz=FuzzSign1
 ```
 
+### Security Reviews
+
+`go-cose` undergoes periodic security review. The security review reports are located [here](https://github.com/veraison/go-cose/tree/main/reports)
```

Comment:
https://github.com/veraison/go-cose/tree/main/reports can be made a relative link (it's inside the same repo)

Reply:
Changes done!